mkdocs.yml

```diff
@@ -81,8 +81,10 @@ nav:
   - Home: index.md
   - AWS:
     - Getting Started: AWS/getting-started-AWS.md
-    - AWS Parallel Cluster Config: AWS/pcluster.md
+      #    - AWS Parallel Cluster Config: AWS/pcluster.md
```

Comment:
@wspear is something broken here? If so perhaps it is from something I did yesterday. Just curious to know more about why you commented this out...

Reply:
AWS/pcluster.md was empty when I checked out the site on July 2nd. The page on the live site was also blank so I figured it was better to remove the link until it was populated.

Reply:
Hmmm let me dig through the git history. I thought we had something here for this.

Reply:
apparently we did not. Thanks Wyatt, sorry for the noise!